rs/types/management_canister_types/src/lib.rs

```diff
@@ -4182,7 +4201,20 @@ pub struct UploadCanisterSnapshotDataArgs {
     pub chunk: Vec<u8>,
 }
 
-impl Payload<'_> for UploadCanisterSnapshotDataArgs {}
+// TODO: EXC-1997.
+impl<'a> Payload<'a> for UploadCanisterSnapshotDataArgs {
```

Comment:
please also follow this pattern here: https://github.com/dfinity/ic/blob/4d2fa64a57ebbbdbf8b68594c80e321e9c3ab4de/rs/types/management_canister_types/src/lib.rs#L3990